docs/changes.txt

```diff
@@ -599,6 +643,16 @@ wxMSW:
 wxOSX:
 
 - Fix build with 10.13 (High Sierra) SDK (François-Xavier Coudert).
+- Fix removing tools from wxToolBar.
+- Fix crash when switching between monitors.
+
+macOS:
```

Comment:
We shouldn't have 3 headers for wxOSX, so this one and `wxOSX` below should be removed.